features/cellular/easy_cellular/EasyCellularConnection.cpp

```diff
@@ -26,9 +26,12 @@
 #include "CellularUtil.h"
 
 #include "EasyCellularConnection.h"
-
 #include "CellularLog.h"
 
+#if MBED_CONF_APP_CELLULAR_USE_APN_LOOKUP || MBED_CONF_PPP_CELL_IFACE_APN_LOOKUP
```

Comment:
For the `MBED_CONF_PPP_CELL_IFACE_APN_LOOKUP`, you're borrowing the existing JSON option from the generic_modem_driver, but presumably that's liable to go. I guess you should have your own option in your own JSON file (defaulting to on as per #6166). Maybe that's what you're aiming at with the other define?

There shouldn't be any references to "MBED_CONF_APP_" defines in an mbed OS component - they would be options for the application's own code defined in its mbed_app.json. Your options should be defined in your mbed_lib.json, and have an appropriate component name prefix rather than the "APP_". So maybe it would be `MBED_CONF_CELLULAR_USE_APN_LOOKUP` for the option `cellular.use_apn_lookup`.

https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/config_system/

Reply:
MBED_CONF_APP_CELLULAR_USE_APN_LOOKUP => MBED_CONF_CELLULAR_USE_APN_LOOKUP 
https://github.com/AriParkkila/mbed-os/commit/9a8f054db207f9403a40b9849f56d3bb0819cda5